ui/imports/shared/controls/TokenDelegateNew.qml

```diff
@@ -0,0 +1,166 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+
+import AppLayouts.Wallet.controls 1.0
+import utils 1.0
+
+StatusListItem {
```

Comment:
Can we please use something more lightweight as the delegate, like `ItemDelegate`? :crying_cat_face:, wdyt?

Reply:
In general yes, I agree, but for now it's important for me that `TokenDelegateNew` has a nice API. The hidden internal below we can refactor later.